text/0000-landing-page.md

```diff
@@ -0,0 +1,58 @@
+# Meta
+[meta]: #meta
+- Name: Create a community landing page
+- Start Date: 1-6-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Prospective contributors to the project don't have a central place to go for information. This RFC proposes the creation of a community or "landing page" repo at `buildpacks/community` with high level information about the project including key personnel, governance, guidelines for contributions, meeting schedules, etc.
+
+# Outcomes
+[outcomes]: #outcomes
+
+ - New contributors feel welcomed.
+ - A new contributor can easily find information they want about the project, can join the slack and WG meetings, and can quickly contribute pull-requests, RFCs, and issues.
```

Comment:
It's been pointed out before that publicly publishing the zoom WG meeting details could be problematic as we currently have no way of moderating anybody with the link participating.